google/cloud/aiplatform/training_jobs.py

```diff
@@ -4138,6 +4146,20 @@ def _model_upload_fail_string(self) -> str:
             "Model."
         )
 
+    @property
+    def evaluated_data_items_bigquery_uri(self) -> Optional[str]:
+        """BigQuery location of exported evaluated examples from the Training Job"""
+
+        self._assert_gca_resource_is_available()
+
+        try:
+            metadata = self._gca_resource.training_task_metadata
+            return metadata["evaluatedDataItemsBigqueryUri"]
```

Comment:
I can't seem to find this field in `gs://google-cloud-aiplatform/schema/trainingjob/definition/automl_time_series_forecasting_1.0.0.yaml`.

Reply:
The yaml in GCS is out-of-date, but the yaml stored in g3 contains this new field.